### DISCUSSION TITLE
Animation editor should default to FPS instead of Seconds

### DISCUSSION THREAD

**Discussion by Kartopod** [godotengine/godot-proposals#10882] [Editor]
To my knowledge, every animation program defaults to frames and it does not make sense that Godot defaults to seconds. It is infinitely more useful to work in FPS and switch to Seconds from time to time for reference, not the other way round. 

One big annoyance as a consequence of this is the fact that you're defaulted to working with an arbitrary snap value with seconds which simply gets in the way and is not even necessary when using FPS.
![image](https://github.com/user-attachments/assets/08b14717-f485-42e8-b354-d3add84ec776)

Also, even if you do set it to FPS, the animation position field at the top left still shows a value in seconds, which is not useful at all. 
![image](https://github.com/user-attachments/assets/e75a25d7-5c4c-4073-862c-f2ee8f15a041)

**Comment by Mickeon** [comment]
Units in time across the entirety of Godot tend to be in seconds, so it's generally fairly consistent as a default.

Does the setting get remembered between editor sessions? I think that should be done, if not.
We could also have the animation position use frames when desired, but keep in mind that due to a multitude of reasons frames are not guaranteed to be exact.

**Comment by fire** [comment]
There's been a series of hacks and interpolation to give the impression that Godot supports seconds in that section, internally it's a rational number like 1/30 where is the 30 is frames per second. So in the project settings that setting is set to an integer like 30.

See https://github.com/godotengine/godot/pull/97569
